src/hotspot/share/gc/shenandoah/shenandoahCollectorPolicy.hpp

```diff
@@ -82,6 +83,10 @@ class ShenandoahCollectorPolicy : public CHeapObj<mtGC> {
   size_t cycle_counter() const;
 
   void print_gc_stats(outputStream* out) const;
+
+  size_t full_gc_count() const {
```

Comment:
Is it worthwhile to harmonize this with tip and rename to `get_fullgc_count()` ?
Unless the idea is to change genshen tip to use `full_gc_count()` like here when you pull these changes down.

Reply:
I went with the style convention established by `cycle_counter`. I don't feel strongly about this. If you prefer `get_full_gc_count`, I can change that here (otherwise, I'll change genshen to use `full_gc_count`).

Reply:
full_gc_count is fine; so long as these eventually get resolved in genshen tip.